### ISSUE TITLE
Label postion for the text inputs

### ISSUE THREAD

**Issue by zhenwan** [carbon-design-system/carbon#1562]
## Detailed description

> Describe in detail the issue you're having.
Accessibility checking tool finds following violation against the **Form** component's Vanilla JS code snippet:

"Text inputs and select elements must have a label before the input control"

The code snippet puts **label**  after **input**( copied from [here](https://www.carbondesignsystem.com/components/form/code))

```
<div class="bx--form-item">
  <input id="text-input-3" type="text" class="bx--text-input" placeholder="Optional placeholder text">
  <label for="text-input-3" class="bx--label">Text Input label</label>
</div>
.......
```

Because of using style lass  "bx--text-input", even above  code is rendered correctly(label is in front of input field after rendered), but it violates  [this rule](https://aat.w3ibm.mybluemix.net/token/43662731-a348-4e58-a8fd-709f21017bc7/cb7d99e6-6f89-4700-bfd2-20e278045a52/archives/2018OctDeploy//doc/w3/help/en-US/idhi_accessibility_check_g1017.html)
 
suggest to move **label** in front of **input** to fix the problem:

```
<div class="bx--form-item">
  <label for="text-input-3" class="bx--label">Text Input label</label>
  <input id="text-input-3" type="text" class="bx--text-input" placeholder="Optional placeholder text">
</div>
.......
```

> Is this issue related to a specific component?
Form
> What did you expect to happen? What happened instead? What would you like to see changed?

> What browser are you working in?
Chrome
> What version of the Carbon Design System are you using?
V9.62.0

**Comment by dakahn** [comment]
[Closing this as resolved](https://www.carbondesignsystem.com/components/form/code)
```html
<div class="bx--form-item">
  <label for="text-input-3" class="bx--label">Text Input label</label>
  <div class="bx--form__helper-text">
    Optional helper text here; if message is more than one line text should wrap (~100 character count maximum)
  </div>
  <input id="text-input-3" type="text"
    class="bx--text-input"
    placeholder="Optional placeholder text">
</div>
```
